### PR TITLE
[DO NOT MERGE] SEO edits for mtp enable

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited title for length.

Edited description to add keyword.

Added note with link to the office-365-atp article.

Edited image alt tag.

@DaniHalfin 

Edits from original PR 2155.

### CODE REVIEW

File: microsoft-365/security/mtp/mtp-enable.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Turn on Microsoft Threat Protection in the Microsoft 365 security center
-description: Learn how to enable Microsoft Threat Protection and start integrating your security incident and response. 
+title: Turn on Microsoft Threat Protection - Microsoft 365 security
+description: Learn how to enable Microsoft Threat Protection (MTP) and start integrating your security incident and response.
```
Comment (lomayor) [inline microsoft-365/security/mtp/mtp-enable.md:3]: Are we allowed to use "MTP" on public-facing text? The description will show up on SERP.

File: microsoft-365/security/mtp/mtp-enable.md [hunk 2]
```diff
@@ -47,7 +51,7 @@ Microsoft Threat Protection aggregates data from the various integrated services
 
 Before you turn on the service, the Microsoft 365 security center ([security.microsoft.com](https://security.microsoft.com)) shows the Microsoft Threat Protection welcome page when you select **Incidents**, **Action center**, or **Hunting** from the navigation pane. These navigation options are not shown if you are not eligible to use Microsoft Threat Protection.
 
-![Image of the Microsoft Threat Protection welcome page shown if Microsoft Threat Protection has not been turned on](../../media/mtp-welcome.png)
+![Image of the Microsoft Threat Protection welcome page shown if feature turned off](../../media/mtp-welcome.png)
```
Comment (lomayor) [inline microsoft-365/security/mtp/mtp-enable.md:54]: if feature is off

**Review by lomayor — CHANGES_REQUESTED** [review]
Please check if changes need to be applied before the merge.

**Comment by DaniHalfin** [comment]
becomes irrelevant post rebrand

**Pull request closed without merging**